mso/resource_mso_schema_site_anp_epg_bulk_staticport.go

```diff
@@ -504,6 +524,8 @@ func resourceMSOSchemaSiteAnpEpgBulkStaticPortUpdate(d *schema.ResourceData, m i
 
 			if staticPortMap["type"] == "port" && static_port_fex != "" {
 				portpath = fmt.Sprintf("topology/%s/paths-%s/extpaths-%s/pathep-[%s]", static_port_pod, static_port_leaf, static_port_fex, static_port_path)
+			} else if staticPortMap["type"] == "vpc" && static_port_fex != "" {
+				portpath = fmt.Sprintf("topology/%s/protpaths-%s/extprotpaths-%s/pathep-[%s]", static_port_pod, static_port_leaf, static_port_fex, static_port_path)
```

Comment:
Would it be possible to move setting the port path string into a function? Looks like its mostly duplicated.
Or alternatively moving the strings into constants.

Reply:
I kept it inline with the current code. 

I think there are plenty of code changes that could be made which would reduce duplication in this resource, and would apply to almost all the other schema related resources. I do not think we should address these rewrites in this PR but should do as separate issues when we decide what the next steps will be for the provider.

Reply:
I agree with @akinross here. A separate PR would be better if we want to bring in radical changes to the resources

Reply:
I had a chat with Akini about this requested change. I am not suggesting to refactor everything to reduce code duplication as this would not be practical. However, I think small incremental refactors around the areas being changed are possible when you are already in context and testing it.
If we are going to be making broader changes to this provider then I am okay to approve.